src/styles.scss

```diff
@@ -503,6 +503,7 @@ $green-font-color: #19B028;
 }
 
 .global__bottom-bar-mobile {
+  z-index: 2;
```

Comment:
do we want the z-index to be the same for the bottom bar and the create post circle?

Reply:
Good point, updated create post circle